tests/unit/test_synapse_api.py

```diff
@@ -87,9 +87,9 @@ def test_generate_mac():
 @mock.patch("synapse.api.requests")
 def test_get_nonce_success(mock_requests):
     """
-    arrange: set User parameters.
-    act: register the user.
-    assert: parameters are passed correctly.
+    arrange: none.
```

Comment:
So there is no pre-condition at all? What about the fact that certain things are mocked?

Reply:
I used Flask PR (https://github.com/canonical/flask-k8s-operator/pull/1) as a reference.

I assumed that arrange would be something related to the charm (deployed, integrated with some database, etc.).
But I can add what is being mocked then.

Reply:
Arrange changed.